lib/provider/camunda/parts/implementation/ProcessVariables.js

```diff
@@ -32,13 +34,13 @@ module.exports = function(element, translate) {
       // title ///////////////////
 
       var collapsible = factory.collapsible({
-        id: scopePrefix + name + '-collapsible',
-        title: name,
+        id: scopePrefix + 'variable- ' + idx + '-collapsible',
```

Comment:
This fixes another, unrelated thing I guess? Or is it just for testing purposes?

Reply:
This actually fixes the original issue. Instead of unescaping the `name`, which would also fix the issue, I found it better to simply use generic ids. Variable names can be very long and cryptic, those ids will be easier to use and maintain.

We actually do the same in other parts, e.g. [input & output parameters](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/lib/provider/camunda/parts/implementation/OutputParameters.js#L155).

Reply:
Makes sense, thanks for clarifying.